mungegithub/mungers/close-stale-pr.go

```diff
@@ -232,11 +231,47 @@ func closePullRequest(obj *github.MungeObject, inactiveFor time.Duration) {
 	obj.ClosePR()
 }
 
+func getInvolvedUsers(obj *github.MungeObject) []string {
+	var users []string
+
+	var assignee string
+	if obj.Issue.Assignee != nil && obj.Issue.Assignee.Login != nil && *obj.Issue.Assignee.Login != "*" {
```

Comment:
No, I'm currently discussing with @eparis about updating the go-github API for that purpose